docs/admin/release_notes/version_1.2.md

```diff
@@ -5,6 +5,12 @@
 - Move GoldenConfigSetting scope to the use of DynamicGroups
 - Drop support of Nautobot <1.4.X
 
+## v1.2.0 - 2022-08
+
+### Fixed
+
+- #314 Fixed Tag filtering not working in job launch form
+
```

Comment:
Can you add to 1.1 as well, what is in 1.1? Because of merge conflicts, can't import it directly. Additionally, can you add any PR's missing?

Reply:
Also, remove todo from this file

Reply:
Done